server/src/test/java/io/crate/execution/engine/join/HashJoinBatchIteratorTest.java

```diff
@@ -52,7 +52,7 @@
 
 @RunWith(RandomizedRunner.class)
 @ThreadLeakScope(ThreadLeakScope.Scope.NONE)
-public class HashInnerJoinBatchIteratorTest {
+public class HashJoinBatchIteratorTest {
```

Comment:
This shouldn't be renamed, right? it's testing only the inner behavior.

Reply:
👍